pkg/driver/controller.go

```diff
@@ -75,16 +76,28 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 
 	// create a new volume
 	zone := pickAvailabilityZone(req.GetAccessibilityRequirements())
+	volumeParams := req.GetParameters()
+	volumeType := volumeParams["type"]
+	iopsPerGB := 0
+	if volumeType == cloud.VolumeTypeIO1 {
+		iopsPerGB, err = strconv.Atoi(volumeParams["iopsPerGB"])
+		if err != nil {
+			return nil, status.Errorf(codes.InvalidArgument, "iopsPerGB is invalid: %v", err)
```

Comment:
nit: since this message is sent to the CO, would it make sense to use a more descriptive error message? Something like:

`Could not parse valid iopsPerGB: %v, err`

Reply:
done